ui/app/src/state/actions/preview.ts

```diff
@@ -175,4 +179,20 @@ export function setItemBeingDragged(active: boolean): StandardAction {
   };
 }
 
+export const fetchPanelAssetsItems = createAction<Partial<ElasticParams>>(FETCH_PANEL_ASSETS_ITEMS);
+
+export function fetchPanelAssetsItemsComplete(searchResult: SearchResult): StandardAction {
+  return {
+    type: FETCH_PANEL_ASSETS_ITEMS_COMPLETE,
+    payload: searchResult
+  }
+}
+
+export function fetchPanelAssetsItemsFailed(error): StandardAction {
+  return {
+    type: FETCH_PANEL_ASSETS_ITEMS_FAILED,
+    payload: error
+  };
+}
```

Comment:
You may use createAction for these too. 

(The mix of usage is mostly due to stuff that was created before we started using redux-toolkit)